test/e2e/split/main.go

```diff
@@ -145,7 +143,7 @@ func createChunkRegexp(numChunks, printChunk int, specs []string) (string, error
 	// Create chunks of size ceil(number of specs/number of chunks) in alphanumeric order.
 	// This is deterministic on inputs.
 	chunks := make([][]string, numChunks)
-	interval := int(math.Ceil(float64(numSpecs) / float64(numChunks)))
```

Comment:
Removal of `math.Ceil()`?
May want to update the comment.

Reply:
I'm going to revert this. Was experimental.